node-operators/hyperdrive/configuration.md

```diff
@@ -39,3 +39,33 @@ With [Grafana](https://grafana.com/) and [Prometheus](https://prometheus.io/) bu
 #### Hyperdrive and TX Fees
 
 Hyperdrive allows users to control manual and automatic transaction settings. In the configuration TUI, you can set the maximum gas, priority fees, and other details to ensure your node operates profitably.
+
+#### Configuration Wizard - Savings Settings
+
+In the last step of the configuration wizard, you can Review All Changes, or simply Save and Exit.  After saving and exiting the wizard, you can start Hyperdrive services.  
+
+`Would you like to start the Hyperdrive services automatically now? [y/n]`
+
+Based on new configuration, Hyperdrive sees that you haven't started the Validator Client before. To be sure, Hyperdrive asks you to confirm there are no active validators. If this is truly a new install on a fresh system, it's safe to say `y` here.
+
+`It looks like this is your first time starting a Validator Client.
+Just to be sure, does your node have any existing, active validators attesting on the Beacon Chain? [y/n]`
+
+There is a final warning, because Hyperdrive can't determine if you may have been attesting in the last 15 minutes. Again, in a fresh install, it's safe to say `y`.
+
+```
+Since your node didn't have any Validator Clients before, Hyperdrive can't determine if you attested in the last 15 minutes.
+If you did, it may resubmit an attestation you have already submitted.
+This will slash your validator!
+To prevent slashing, you must wait 15 minutes from the time you stopped the clients before starting them again.
+
+Press y when you understand the above warning, have waited, and are ready to start Hyperdrive: [y/n]
+```
```

Comment:
Best to move the output text above the explanation (I think it's reversed right now)

Reply:
Updated